<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelsForm</class>
 <widget class="QWidget" name="ChannelsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/channels</normaloff>:/icon/channels</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="retuneButton">
     <property name="text">
      <string>Rescan</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableView" name="table"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="frequencyCheckBox">
     <property name="text">
      <string>Show frequency column</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
